.auditrc.json

```diff
@@ -1,8 +1,9 @@
 {
```

Comment:
when running locally cmd `npm run audit:test` will use this file to accept vul from the allowList.